### PR TITLE
Remove reporting feature from state extraction

### PR DESCRIPTION
**Pull request by zhangchiqing**
This PR removes the reporting feature for the state extraction util, why?

because:
- The report takes a really long time, and causing it’s hard to tell if the state extraction is completed
- The [post reporting](https://github.com/onflow/flow-go/blob/master/cmd/util/cmd/execution-state-extract/execution_state_extract.go#L109) can be done with a different command line, it just needs to read the checkpoint file again into memory, which is now quite fast.
- The [pre reporting](https://github.com/onflow/flow-go/blob/master/cmd/util/cmd/execution-state-extract/execution_state_extract.go#L101) is currently used printing epoch counter, which we are no longer using during spork, instead we are sending a script querying on an EN to find the latest epoch counter, and using the epoch end view to determine whether the last sealed block exceeds the epoch boundary.
- We can also remove the --chain parameter from the state extraction util, which is only needed by the reporting feature.

This PR also speeds up the state extraction.

### CODE REVIEW

File: ledger/complete/ledger.go [hunk 1]
```diff
@@ -462,25 +443,6 @@ func (l *Ledger) ExportCheckpointAt(
 
 	l.logger.Info().Msgf("checkpoint file successfully stored at: %v %v", outputDir, outputFile)
 
-	l.logger.Info().Msgf("start running post-checkpoint reporters")
-
-	if noMigration {
-		// when there is no mgiration, we generate the payloads now before
-		// running the postCheckpointReporters
-		payloads = newTrie.AllPayloads()
```
Comment (zhangchiqing) [inline ledger/complete/ledger.go:470]: If there is no migration, we are now skipping this time consuming call. Will save some spork time

**Review by peterargue — APPROVED** [review]
looks good. is there already a separate command to generate the reports?

**Comment by zhangchiqing** [comment]
>is there already a separate command to generate the reports?
no, the report will be done in a separate PR

File: cmd/util/cmd/execution-state-extract/cmd.go [hunk 2]
```diff
@@ -135,19 +124,20 @@ func run(*cobra.Command, []string) {
...
 	}
 
-	err = extractExecutionState(
+	err := extractExecutionState(
 		flagExecutionStateDir,
 		stateCommitment,
 		flagOutputDir,
 		log.Logger,
-		chain,
 		!flagNoMigration,
```
Comment (janezpodhostnik) [inline cmd/util/cmd/execution-state-extract/cmd.go:140]: I think no migration is also redundant now, right?

**Review by janezpodhostnik — APPROVED** [review]
LGTM.
Does an issue to add a new command just for reporters already exist?

**Comment by zhangchiqing** [comment]
issued up https://github.com/onflow/flow-go/issues/4418

**Comment by zhangchiqing** [comment]
bors merge

**Pull request merged**